employees/forms.py

```diff
@@ -48,7 +49,10 @@ def clean(self, value: str) -> str:
         return f"{hours}:{minutes}:00"
 
     def prepare_value(self, value: str) -> str:
-        return f"{value}:00"
+        if isinstance(value, datetime.timedelta):
+            return duration_string(value)
+        else:
+            return f"{value}:00"
```

Comment:
What is happening here? When the value isn't considered a timedelta?

Reply:
Because this value is passed to form in html so it need to be string :)

Reply:
Ok, so I understand that the error only occurs when we're writing in full hours, since it's not recognised as the timedelta? Hence the `f"{value}:00"`